### DISCUSSION TITLE
How to simulate curl error

### DISCUSSION THREAD

**Discussion by huantingc** [mitmproxy/mitmproxy#6802] [Q&A]
Hi, 
I want to intercept request and return response by proxy.
Is it possible to simulate curl error by writing addons?

**Comment by Prinzhorn** [comment]
> I want to intercept request and return response by proxy.

https://github.com/mitmproxy/mitmproxy/blob/b858a826bf20a328790b3a3da2320e6d726f97b6/examples/addons/http-reply-from-proxy.py

> Is it possible to simulate curl error by writing addons?

No idea what that means.
